ESPAsyncWiFiManager.cpp

```diff
@@ -496,16 +497,20 @@ boolean  AsyncWiFiManager::startConfigPortal(char const *apName, char const *apP
       WiFi.disconnect (false);
     #endif
       scan();
-      if(_tryConnectDuringConfigPortal) WiFi.begin(); // try to reconnect to AP
+      if(_tryConnectDuringConfigPortal) {
+		  WiFi.begin(); // try to reconnect to AP
+		  connectedDuringConfigPortal = true;
```

Comment:
Reconnection has been successful here. Mark flag true to avoid callback to be called